dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_9__Add_last10years_column_into_relativeperiods_table.sql

```diff
@@ -0,0 +1 @@
+alter table public.relativeperiods add column if not exists last10years boolean not null default false;
```

Comment:
We have a convention of not using default values. Instead 1) add column as nullable 2) set null values to false 3) add not null constraint to column.